packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -171,8 +181,8 @@ export default mixins(showMessage, nodeHelpers).extend({
 			required: true,
 		},
 		activeId: {
-			type: [String],
-			required: true,
+			type: [String, Number],
```

Comment:
When can the ID be a number? I thought we switched to strings entirely.

Reply:
Looks like nowhere. Using only String